test/ex_doc/formatter/html_test.exs

```diff
@@ -101,4 +119,56 @@ defmodule ExDoc.Formatter.HTMLTest do
     assert HTML.pretty_codeblocks(iex_detected_without_class) ==
           "<pre><code class=\"iex elixir\">iex&gt; max(4, 5)"
   end
+
+  test "run generates the redirect index.html" do
+    config = %ExDoc.Config{output: "#{output_dir}/redirect", main: "RandomError", }
+    HTML.run(get_modules(config), config)
+
+    assert File.regular?("#{output_dir}/redirect/RandomError.html")
+    assert File.regular?("#{output_dir}/redirect/index.html")
+    content = File.read!("#{output_dir}/redirect/index.html")
+    assert content =~ ~r{<meta http-equiv="refresh" content="0; url=RandomError.html"\s*/>}
```

Comment:
Does it make sense to do an assert on `File.regular?(file)` when right after we are doing an assert on the content of that file?